### PR TITLE
Expedition

### PR DESCRIPTION
**Pull request by wadie**
Expedition use case.

We might still need the images for it

### CODE REVIEW

File: src/ui/components/ExpeditionCaseThumb/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-blend-mode: soft-light;
+}
+
+.breethe-right-background {
+  background-color: resolve("shape-right.background");
+  background-color: #6f42c1;
+}
```
Comment (marcoow) [inline src/ui/components/ExpeditionCaseThumb/stylesheet.css]: 👀

**Review by marcoow — CHANGES_REQUESTED** [review]
`package-lock.json` should not be added - we use yarn and have `yarn.lock` already

**Comment by ghislaineguerin** [comment]
@wadie Do you have an issue where the images are 404 on stage?

**Comment by wadie** [comment]
@ghislaineguerin only the images that I haven't set yet

Screenshot_2019-03-26 Hello Glimmer

**Comment by ghislaineguerin** [comment]
@wadie can you add placeholder images to verify the layout and I'll replace them later?

**Comment by wadie** [comment]
updated

**Comment by ghislaineguerin** [comment]
@wadie the bottom image should remain the same, it's supposed to be a link to another case study so the color should remain the same. Other than that, I'd replace the quote under client with the actual client description. Thanks

**Comment by ghislaineguerin** [comment]
@wadie unrelated, but when clicking on a footer link why doesn't the page jump back to top?

**Comment by wadie** [comment]
Yeah the route changes but the focus stays on the footer and it's happening with all the pages/links.

I think it's an unrelated issue, so we can merge this PR then I'll create a new branch to fix it.

**Review by ghislaineguerin — APPROVED** [review]

**Pull request merged**